builder_pattern/house.go

```diff
@@ -0,0 +1,23 @@
+package builder_pattern
+
+// House : we can also give House interface to client
+// and make some fields hidden if we want to
+// like architectural design logic. And only
+// expose required fields which are of in interest
+// to client
+type House struct {
```

Comment:
House is the object that is needed to be build right?
If yes, let's please add in the documentation?